tests/properties.json

```diff
@@ -70,7 +70,7 @@
       "heartbeat": "/",
       "notify_path": "/notify",
       "notify_port": "18666",
-      "port": "18666",
+      "port": "8666",
```

Comment:
that's Ok it's just config, it may be adapted to the specific target under tests